### PR TITLE
telco5g: Run CNF tests in serial mode

### PR DESCRIPTION
**Pull request by sshnaidm**
Disable parallel testing and run all in serial.
[CNF-7150](https://issues.redhat.com//browse/CNF-7150)

### CODE REVIEW

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 1]
```diff
@@ -135,13 +122,24 @@ function is_bm_node {
     return 1
 }
 
+function create_ns {
```
Comment (sabinaaledort) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:125]: no used

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 2]
```diff
@@ -294,100 +213,36 @@ if [[ "$CNF_BRANCH" == *"4.13"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
...
-    done
-done
-wait
-
-# cleanup
-for node in ${test_nodes}; do
-    rm -f "${node}_ready.txt"
-done
-rm -rf node/
+export ROLE_WORKER_CNF="worker-cnf"
```
Comment (sabinaaledort) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:216]: that's the default label so no need to set it

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 3]
```diff
@@ -294,100 +213,36 @@ if [[ "$CNF_BRANCH" == *"4.13"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
...
-    xml_v="${ARTIFACT_DIR}/${feature}/validation_junit.xml"
-    if [[ -f $xml_v ]]; then
-        cp $xml_v ${ARTIFACT_DIR}/validation_junit_${feature}.xml
-    fi
-    xml_s="${ARTIFACT_DIR}/${feature}/setup_junit.xml"
-    if [[ -f $xml_s ]]; then
-        cp $xml_s ${ARTIFACT_DIR}/setup_junit_${feature}.xml
-    fi
-done
+export CNF_NODES="${test_nodes}"
```
Comment (sabinaaledort) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:240]: if you are using all nodes, no need to set it

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh (hunk 2 above)
Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:216]: ack

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh (hunk 1 above)
Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:125]: ack

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh (hunk 3 above)
Comment (sabinaaledort) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:240]: oh i thought you removed it, i didn't see it in the artifacts

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:240]: I tried w/o it and w/ it, not sure if we need it though. Trying to get the answer :)

**Pull request merged**